chimera/pars/R3BAsyChimeraContFact.cxx

```diff
@@ -0,0 +1,77 @@
+// ----------------------------------------------------------------------
+//								    -----
+//  Factory for the parameter containers in libR3BAsyChimera           -----
+//         Created 03/11/19  by J.L. Rodriguez-Sanchez              -----
```

Comment:
Please, update date and author

Reply:
Done. We have preferred to update authors and date in .h (header file)    
keeping 
// -------------------------------------------------------------------------
// -----                        R3BAsyxxxxx source file            -----
// -------------------------------------------------------------------------
in cxx source files as seen for other R3Broot devices